pennylane/devices/default_qutrit.py

```diff
@@ -93,6 +95,31 @@ class DefaultQutrit(QutritDevice):
         "Identity",
     }
 
+    # Static methods to use qml.math to allow for backprop differentiation
+    _reshape = staticmethod(qnp.reshape)
+    _flatten = staticmethod(qnp.flatten)
+    _transpose = staticmethod(qnp.transpose)
+    _dot = staticmethod(qnp.dot)
+    _stack = staticmethod(qnp.stack)
+    _conj = staticmethod(qnp.conj)
+    _roll = staticmethod(qnp.roll)
+    _cast = staticmethod(qnp.cast)
+    _real = staticmethod(qnp.real)
+    _imag = staticmethod(qnp.imag)
+
+    @staticmethod
```

Comment:
Reminder: leave comment here to explain where these functions came from / why they are needed